frontend/public/module/k8s/selector-requirement.js

```diff
@@ -8,42 +8,35 @@ export const createEquals = (key, value) => ({
   values: [value],
 });
 
+const validateKey = (key, operator, values) => {
+  if (/^(([A-Za-z0-9][-A-Za-z0-9.]*)?[A-Za-z0-9])$/.test(key)) {
+    return { key, operator, values };
```

Comment:
I still believe this method should return a boolean, not an object.

Reply:
No problem, I will make the changes.